notes.txt

```diff
@@ -11,3 +11,11 @@ git pull
 '''
```

Comment:
Hi, you can merge this into master.
For everyday code check-in, please share pull request in the email. Thanks